### ISSUE TITLE
datalad run should allow for an empty folder as output

### ISSUE THREAD

**Issue by yarikoptic** [datalad/datalad#7653]
```
❯ datalad create test
create(ok): /tmp/test (dataset)

❯ cd test

❯ datalad run --output output ls '{outputs}'
[INFO   ] == Command start (output follows) ===== 
ls: cannot access 'output': No such file or directory
[INFO   ] == Command exit (modification check follows) ===== 
[INFO   ] The command had a non-zero exit code. If this is expected, you can save the changes with 'datalad save -d . -r -F .git/COMMIT_EDITMSG' 
run(error): /tmp/test (dataset) [ls 'output']

❯ mkdir output

❯ datalad run --output output ls '{outputs}'
status(error): output [File unknown to git]
❯ datalad --version
datalad 1.1.0
```

Discovered by @asmacdo .  IMHO datalad must not error out -- it should pretty much ignore empty folders among outputs since those are not tracked by `git`.

**Comment by asmacdo** [comment]
Workaround: just add `.placeholder` and datalad save prior to `datalad run`

**Comment by yarikoptic-gitmate** [comment]
Issue fixed in [`1.1.4`](https://github.com/datalad/datalad/releases/tag/1.1.4)
